src/components/PersonInformationForm/PersonInformationForm.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
```

Comment:
Tbh I don't get how it's going to work. Like how would you use this data to calculate user's workout results? 🤔

Reply:
The task was about creating the form. So I if it is looking good, I can start thinking about fetching data from this form to use in calories calculation